### ISSUE TITLE
Passing elementId to the render function not working on dev

### ISSUE THREAD

**Issue by anchovy** [solspace/craft-freeform#427]
**Describe the bug or issue you're experiencing**
We've got a form that's updating users details - this is working as expected locally, but in our dev environment with the same codebase (develop branch) and database, it's not working and there's no error messages in the logs at all.

I'm displaying the user's id above the form to ensure it's picking it up correctly and it is.

```
{{currentUser.id}}
{{ form.render({elementId: currentUser.id}) }}
```

(local on the left, dev on the right)
![image](https://user-images.githubusercontent.com/599005/184730374-53af1f8c-3226-4680-864f-5278fffaecf3.png)

Any ideas?

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 3.7.51
- Freeform Version: 3.13.17
- Freeform Edition: Pro
- Fresh Install or Upgrade: N/A

**Comment by kjmartens** [comment]
Hi @anchovy,

Out of curiosity, in the dev branch, do you set up the Element Connections feature and field mapping for the form as you did in the local environment? 🙂

**Comment by anchovy** [comment]
Hi @kjmartens 

Yep - it's using the same database & settings. The form & settings was actually added on dev and then pulled back to my local.

**Comment by kjmartens** [comment]
@anchovy Are you able to copy and paste the JSON code of the affected form on local here or in a private support ticket (`layoutJson` column in `freeform_forms` database table)? 🙂

**Comment by anchovy** [comment]
Thanks @kjmartens - support ticked with the the `layoutJson` data sent.

**Comment by anchovy** [comment]
Hi @kjmartens did you get that json data?

Of note - if I do complete the blank form and submit it does update the correct User entry with the associated field mapping from the form.... super strange.

**Comment by kjmartens** [comment]
Hi @anchovy, I just reviewed it now. Seems like it's all there. I wonder if there's anything else out of place... can you share the full template code with me and also make sure both environments are using the same formatting template, etc. 🙂

**Comment by anchovy** [comment]
Hi @kjmartens - thanks, just updated my comment above. It's using the default form rendering tag.

**Comment by anchovy** [comment]
@kjmartens - debugging further it's correctly getting the elementId in `ElementEditBundle::getElementId()` and setting the values in `ElementEditBundle::populateFormWithElementValues()`

**Comment by anchovy** [comment]
Hi @kjmartens 

I've added some further debugging code and now I'm getting really confused. It seems to be setting the fields values correctly based on the supplied elementId (and I can pass override values to the form and these get rendered) but when the field is rendered the values are not coming through.

**Render the form with some override values**
```
{% set overrideValues = { 
        "firstName": 'Forced first name',
        "initial": 'Forced Initial',
        "spouse": 'Forced Spouse'
    }
%}
{{ form.render({
    elementId: currentUser.id,
    overrideValues: overrideValues
}) }}
```


**Setting the value**
File: vendor/solspace/craft-freeform/packages/plugin/src/Library/Composer/Components/Fields/Traits/SingleValueTrait.php
```
public function setValue($value)
{
        $this->value = $value;

        # DEBUG
        if ( is_string($value) && $value != '') echo $this->handle.': '. ($value).' ('.$this->getValue() .') <br />';
        

```
**Getting the value**
File: vendor/solspace/craft-freeform/packages/plugin/src/Fields/TextField.php

```
  protected function getInputHtml(): string
  {
        # DEBUG
        var_dump($this->getValue());
```

As far as I can see, the getInputHtml function uses the SingleValueTrait to get the value which when setting the value works but rendering the field in the same page load/session now empty, unless it's been overwritten?

<img width="870" alt="image" src="https://user-images.githubusercontent.com/599005/185515085-1573c0f7-bfb7-418f-a4c1-e1c845317b74.png">


Any further ideas?

**Comment by anchovy** [comment]
@kjmartens I've rolled back through the plugin versions and on version 3.12.11 the values render as expected. v3.12.12 onwards they don't.

The change log for 3.12.12 has the following:
 _- Fixed a bug where fields with default values specified in the form builder would not show the default value/option on multi-page forms after the first page._

Could this have something to do with it?

**Comment by anchovy** [comment]
@kjmartens - can you see anything that changed in v3.12.12 that might be causing this issue?

**Comment by kjmartens** [comment]
Sorry for the trouble, and thanks for the PR @anchovy . We'll review and add it soon. 🙂

**Comment by anchovy** [comment]
Hi @gustavs-gutmanis - any ideas when pull request #439 will be merged in and released as stable?

**Comment by kjmartens** [comment]
Hi @anchovy,

This is now available in Freeform 3.13.19 and 4.0.3. 🙂
